TwitchChannelPointsMiner/classes/entities/Chat.py

```diff
@@ -0,0 +1,18 @@
+import irc.bot
+
+
+class Chat(irc.bot.SingleServerIRCBot):
+    def __init__(self, username, token, channel):
+        self.token = token
+        self.channel = "#" + channel
+
+        # Create IRC bot connection
+        server = "irc.chat.twitch.tv"
+        port = 6667
+        irc.bot.SingleServerIRCBot.__init__(
```

Comment:
- `server = "irc.chat.twitch.tv"`
- `port = 6667`

Should be moved in `constants/twitch.py`